core/toolbox.js

```diff
@@ -533,7 +533,9 @@ Blockly.Toolbox.TreeNode = function(toolbox, html, opt_config, opt_domHelper) {
   goog.ui.tree.TreeNode.call(this, html, opt_config, opt_domHelper);
```

Comment:
I hope you don't mind me asking a general question here: why is TreeNode a "class function" rather a "member method"? Is it because it extends `goog.ui.tree.TreeNode`?